getting_started/setup_linux/setup_ktest_environment.rst

```diff
@@ -331,50 +333,51 @@ Save the following in sof-dev.conf.
   BUILD_NOCLEAN = 1
 
 
-For Fedora and derived distributions, make the following changes:
+For targets running Fedora and derived distributions, make the following changes:
 
 .. code-block:: perl
 
-  GRUB_MENU    = "title" of the kernel entry as displayed by: 'grubby --info=ALL'
+  GRUB_MENU    = ${LOCALVERSION}
   GRUB_REBOOT  = grub2-reboot
   REBOOT_TYPE  = grub2bls
-  POST_INSTALL = ssh  -o 'ProxyCommand none' $SSH_USER@$MACHINE sudo dracut --hostonly --force --kver ${LOCALVERSION}
+  POST_INSTALL = ssh  -o 'ProxyCommand none' $SSH_USER@$MACHINE sudo dracut --hostonly --force /boot/initramfs-${LOCALVERSION}.img $KERNEL_VERSION
```

Comment:
Can you please take the opportunity to remove all the `-o 'ProxyCommand none'` occurrences? They really don't belong to this guide. People who need this must put it once for all in .ssh/config, certainly not in their ktest config file.